include/boost/winapi/process.hpp

```diff
@@ -47,15 +47,18 @@ struct _STARTUPINFOEXW;
 
 #if BOOST_WINAPI_PARTITION_DESKTOP_SYSTEM
 
-BOOST_SYMBOL_IMPORT BOOST_NORETURN boost::winapi::VOID_ BOOST_WINAPI_WINAPI_CC
+#if !defined(_WIN32_WCE)
+// On Windows CE ExitProcess is a macro to call TerminateProcess
```

Comment:
In this case, Boost.WinAPI has to provide a portable way to call `ExitProcess`, including in cases when it is a macro and when it is not defined. The best approach is to provide an inline wrapper function in `boost::winapi` (e.g. `exit_process`).

Reply:
agree